ReleaseHistory.md

```diff
@@ -20,6 +20,8 @@
 
 ## v4.3.8 UNRELEASED
 - BRK: `ValidationResult` constructor now sets `ValidationState` to `Unknown`. [#733](https://github.com/microsoft/sarif-pattern-matcher/pull/733)
+- BRK: `ValidatorBase.ReturnUnhandledException` now requires an `asset` parameter. [#736](https://github.com/microsoft/sarif-pattern-matcher/pull/736)
```

Comment:
>BRK [](http://example.com/codeflow?start=2&length=3)

We add a release note both for the tool consumer (who sees improved error reporting for this case) and for the API consumer (who sees a breaking change in the now required parameter). This is typical for the release notes: different audiences may observe the change in compile or runtime behavior.